apps/src/JSInterpreter.js

```diff
@@ -45,6 +45,7 @@ var JSInterpreter = module.exports = function (options) {
   this.paused = false;
   this.yieldExecution = false;
   this.startedHandlingEvents = false;
+  this.executionError = null;
```

Comment:
I'm not sure I like moving this out here (vs. passing it as `err`).  It's one more piece of state we'll need to make sure gets cleared correctly.  When the program is run a second time, won't this still have the old error value?